backends/scg/scg.c

```diff
@@ -1,40 +1,55 @@
+#include <errno.h>
+#include <stdio.h>
+#include <stdlib.h>
 #include <X11/Xlib.h>
-#include <X11/Xproto.h>
-#include <X11/Xatom.h>
 #include <X11/extensions/Xrandr.h>
 
-#include <stdlib.h>
-#include <math.h>
+char *argv0 = "scg";
 
-int main(int argc, char **argv) {
-    Display *dpy = XOpenDisplay(NULL);
+#define usage() fprintf(stderr, "usage: %s GAMMA_R GAMMA_G GAMMA_B\n", argv0)
```

Comment:
This is not meant for interactive use, so I am not sure if this really serves a purpose.

Reply:
Same as above.